python/pyspark/accumulators.py

```diff
@@ -46,7 +48,10 @@ def _deserialize_accumulator(aid, zero_value, accum_param):
         return accum
 
 
-class Accumulator(object):
+T = TypeVar("T")
```

Comment:
Let's not touch things which are not strictly related to this particular PR. Not to mention we should never have annotations in both `py` and `pyi` files.